go/check-logs-sequence/samples.log

```diff
@@ -0,0 +1,18 @@
+{"message": "This is testing E0412 sample test"}
```

Comment:
Those are very short logs lines ... can you add the long lines as we got them from the customer (but obfuscated if needed) ... so this will simulate really the length of the lines and will represent better real-life scenarios ... we are checking performance so this is needed